src/main/java/de/tum/in/www1/artemis/repository/StudentQuestionAnswerRepository.java

```diff
@@ -12,4 +13,8 @@
 @Repository
 public interface StudentQuestionAnswerRepository extends JpaRepository<StudentQuestionAnswer, Long> {
 
+    default StudentQuestionAnswer findByIdElseThrow(long studentQuestionId) {
```

Comment:
you could also add `@NotNull` here, as you did in the other repositories

```suggestion

    @NotNull
    default StudentQuestionAnswer findByIdElseThrow(long studentQuestionId) {
```